osu_media.post_update.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file
+ * Post Update file.
+ */
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Remove view mode for media_card_caption.
+ */
+function osu_media_post_update_remove_card_caption_display(&$sandbox) {
+  /** @var \Drupal\Core\Config\Config $media_card_caption_image_display */
+  $media_card_caption_image_display = \Drupal::service('config.factory')
+    ->getEditable('core.entity_view_display.media.image.media_card_caption');
+  $media_card_caption_image_display->delete();
```

Comment:
Some whitespace between each section here would make it much easier to read

Reply:
Added some space around the code.